modules/monitor/core/metrics/metricq/provider.go

```diff
@@ -20,8 +20,8 @@ import (
 	"github.com/erda-project/erda-infra/base/logs"
 	"github.com/erda-project/erda-infra/base/servicehub"
 
-	//"github.com/erda-project/erda-infra/providers/httpserver"
-	//"github.com/erda-project/erda-infra/providers/httpserver/interceptors"
+	// "github.com/erda-project/erda-infra/providers/httpserver"
+	// "github.com/erda-project/erda-infra/providers/httpserver/interceptors"
```

Comment:
delete it if you don’t need it

Reply:
Reserve for test verify that the migration function is intact